zeppelin-web/src/app/jobmanager/jobmanager.controller.js

```diff
@@ -32,14 +32,13 @@ angular.module('zeppelinWebApp')
         $scope.JobInfomationsByFilter = $scope.jobInfomations;
 
         websocketMsgSrv.getNotebookJobsList();
-        var refreshObj = $interval(function() {
-          if ($scope.lastJobServerUnixTime !== undefined) {
-            websocketMsgSrv.getUpdateNotebookJobsList($scope.lastJobServerUnixTime);
-          }
-        }, 1000);
+        // var refreshObj = $interval(function() {
+        //   if ($scope.lastJobServerUnixTime !== undefined) {
+        //     websocketMsgSrv.getUpdateNotebookJobsList($scope.lastJobServerUnixTime);
+        //   }
+        // }, 1000);
 
         $scope.$on('$destroy', function() {
-          $interval.cancel(refreshObj);
           websocketMsgSrv.unsubscribeJobManager();
```

Comment:
To receive the Event JOB, you must register the subscription of the user's browser.
When a user opens the Job Menu, which is registered in the server automatically.
When leaving the Job Menu, you will not receive the Job event unsubscribe.